lib/unleash/toggle_fetcher.rb

```diff
@@ -126,5 +127,23 @@ def read!
         file&.close
       end
     end
+
+    def bootstrap
+      unleash_response = Unleash::Bootstrap::Handler.new(Unleash.configuration.bootstrap_config).retrieve_toggles
+      synchronize_with_local_cache! get_features unleash_response
```

Comment:
So that the variable is more descriptive:
```suggestion
      bootstrap_payload = Unleash::Bootstrap::Handler.new(Unleash.configuration.bootstrap_config).retrieve_toggles
      synchronize_with_local_cache! get_features bootstrap_payload
```